dandi/metadata.py

```diff
@@ -125,7 +125,12 @@ def extract_age(metadata):
             return ...
     else:
         duration = timedelta2duration(start - dob)
-    return models.PropertyValue(value=duration, unitText="Years from birth")
```

Comment:
As a duration can be a mixture of units, it might be better to just always set the `unitText` to "Time from birth".

Reply:
so it wouldn't really contain any form of "Unit" itself. Any reason why we use `unitText` ?